cmd/runtimetest/main.go

```diff
@@ -204,6 +204,39 @@ func validateSysctls(spec *specs.LinuxSpec, rspec *specs.LinuxRuntimeSpec) error
 	return nil
 }
 
+func validateMount(spec *specs.LinuxSpec, rspec *specs.LinuxRuntimeSpec) error {
+	fmt.Println("validating mount")
+	//read the /proc/mount file and covert to map[path]spec.mount
```

Comment:
@wking  thank you for your advice!
1. about required mounts check, I think it's  more appropriate to finish it in bunlde validator , not in runtime test. That means if you do not load these required mounts, your bundle is considered `illegal` 
2. I agree with your idea about not validating any bundle's mounts, but there is a  problem that in current testing framework , we use a single bundle to test all `specs` items , so we need  to cover the restrictions defined in `specs`, like `type` in mounts [1]

[1]https://github.com/opencontainers/specs/blob/master/runtime-config.md